src/http/server.ts

```diff
@@ -684,7 +716,7 @@ export class Server {
    * @return void
    */
   protected async executeMiddlewareServerLevelBeforeRequest(
-    request: any,
+    request: Drash.Http.Request,
```

Comment:
Has the doc block been updated to reflect this?

Reply:
3565ceb